OrbitGl/CallStackDataView.h

```diff
@@ -27,8 +27,8 @@ class CallStackDataView : public DataView {
   void OnContextMenu(const std::string& action, int menu_index,
                      const std::vector<int>& item_indices) override;
   void OnDataChanged() override;
-  void SetCallStack(const CallStack& callstack) {
-    callstack_ = std::make_unique<CallStack>(callstack);
+  void SetCallStack(const CallStack* callstack) {
```

Comment:
Do we ensure, that this won't become nullptr?

Reply:
It becomes nullptr only if we change callstack_data, but we should clear CallstackDataView when changing the callstack_data